Dockerfile

```diff
@@ -47,7 +47,7 @@ COPY install-miniforge.bash /tmp/install-miniforge.bash
 RUN /tmp/install-miniforge.bash
 
 COPY conda-linux-64.lock /tmp/conda-linux-64.lock
-RUN conda install --name base --file /tmp/conda-linux-64.lock
+RUN --mount=type=cache,target=/opt/conda/pkgs conda install --name base --file /tmp/conda-linux-64.lock
```

Comment:
There are other tips from @xhochy's post that may be useful. I believe I added a link in the dockerfile.

Like removing the static linked files (.a).

Reply:
Ya, I'm going to play with more of those. I ran out of steam around midnight, and wanted to capture the big win.